lsp-positions/Cargo.toml

```diff
@@ -18,9 +18,11 @@ test = false
 
 [features]
 default = ["tree-sitter"]
+bincode = ["dep:bincode"]
 
 [dependencies]
 memchr = "2.4"
 tree-sitter = { version=">= 0.19", optional=true }
 unicode-segmentation = { version="1.8" }
 serde = { version="1", optional=true, features=["derive"] }
+bincode = { version="2.0.0-rc.3", optional=true }
```

Comment:
Would it be possible to depend on the latest stable version? I get the impression a 2.0.0 release doesn't have a target date yet. If 1.3.3 is used with a litte endian, varint configuration, it should be equal to the v2 [default config](https://docs.rs/bincode/2.0.0-rc.3/bincode/config/fn.standard.html), if I understand it correctly. [migration details](https://github.com/bincode-org/bincode/blob/trunk/docs/migration_guide.md).